### DISCUSSION TITLE
A book about Large Language Models is coming!

### DISCUSSION THREAD

**Discussion by peremartra** [peremartra/Large-Language-Model-Notebooks-Course#17] [Announcements]
I'm very happy to announce that the repository will be the cornerstone of a book published by Apress. At the moment, the technical editor has already reviewed the first four chapters and it seems they were very well received! So officially, the book will be published in 2024! On my part, I have committed to delivering all the chapters by the end of July and so far I am on time. So I suppose it will be available this summer.

Thank you very much to everyone who has given a star to the repository, and especially to those who have contributed with your opinions and your Pull Requests!

**Comment by rajeck** [comment]
Awesome - I have learned a lot by running through all your examples and look forward to your book.

**Reply by peremartra** [reply]
Thanks rajeck! 

All chapter are delivered. Now it is time for review the Technical Editor suggestions. 

Thanks for your support!I'll try to raffle a book among the star-gazers of the repository.

**Comment by Temitope-Emmanuel** [comment]
Awesome work Pere, so happy for you

**Comment by peremartra** [comment]
![979-8-8688-0514-1](https://github.com/user-attachments/assets/2ae714fe-8cd0-4c67-a77a-10af45b076f4)
The cover is Here!

But we need to wit some weeks for the book, maybe end september.

**Reply by peremartra** [reply]
Available for preorder!

https://amzn.to/3X1mjJB

I hope it will be available in mid-September!
